packages/api/endpoints/granules.js

```diff
@@ -940,6 +949,159 @@ async function del(req, res) {
   return res.send({ detail: 'Record deleted', ...deletionDetails });
 }
 
+const bulkChangeCollectionSchema = z.object({
+  sourceCollectionId: z.string().nonempty('sourceCollectionId is required'),
+  targetCollectionId: z.string().nonempty('targetCollectionId is required'),
+  batchSize: z.number().positive().optional().default(100),
+  concurrency: z.number().positive().optional().default(100),
```

Comment:
How were the default batchSize and concurrency determined?

Reply:
Mostly via discussion with @etcart  and/or @Nnaga1  as the earlier parts of the feature were developed.   We need to get the other tasks in and do a bit of load testing before acceptance/release, and these may need to be modified.

Reply:
Fair enough. If there's gonna be testing and tweaking once it's all put together 👍 

And really the only dealbreaker would be if the defaults don't work. If they're too aggressive so as to cause problems. Which... it doesn't seem like this is too aggressive.